### ISSUE TITLE
Can't sign in 

### ISSUE THREAD

**Issue by ghost** [vauvenal5/yaga#78]
https://youtu.be/XdXFpFOlhpA

My specs:
NextCloud 20.0.4
Raspberry Pi 4B 4GB

Phone: 
Android 11
SD855
MIUI 21.1.20

**Comment by vauvenal5** [comment]
Thx, for the but report. Not sure why the `Open in Browser` button does not work. As for the internal webview I guess this might be a Flutter webview issue. I will check both out.

**Comment by ctgcwiqc** [comment]
I'm having same issue using android11 on pixel3a and on fire tablet with gapps installed.

**Comment by vauvenal5** [comment]
@ctgcwiqc are you using a self-signed certificate? Can you try how it behaves if you add the port?

**Comment by vauvenal5** [comment]
@BobVolskiy I just watched your screen capture once more and noticed that you are using a self-signed cert. This is the actually issue here. Yaga does not yet support self-singed certs, see #40.

**Comment by ctgcwiqc** [comment]
Yes in using self signed cert.  i will import and test.  Thanks.  May be good idea to document this in app description.

Sent from Yahoo Mail on Android 
 
  On Sat, Feb 6, 2021 at 7:19 AM, vauvenal5<notifications@github.com> wrote:   


@BobVolskiy I just watched your screen capture once more and noticed that you are using a self-signed cert. This is the actually issue here. Yaga does not yet support self-singed certs, see #40.

**Comment by ctgcwiqc** [comment]
I purchased a domain from godaddy and set up SSL via letsencrypt and now can sign in.  I do think lots of people would like the ability to use self-signed certs so they do not have to purchase domain name however i paid for a domain for 10 years and it was only 90USD.  Now I have legitimate free SSL cert and no annoying browser warnings!
    On Saturday, February 6, 2021, 07:19:13 AM CST, vauvenal5 <notifications@github.com> wrote:  
 
 


@BobVolskiy I just watched your screen capture once more and noticed that you are using a self-signed cert. This is the actually issue here. Yaga does not yet support self-singed certs, see #40.
